src/snyk/extension.ts

```diff
@@ -441,39 +439,25 @@ class SnykExtension extends SnykLib implements IExtension {
       configuration,
     );
 
-    const codeScansViaLs = await this.experimentService.isUserPartOfExperiment(
-      ExperimentKey.CodeScansViaLanguageServer,
-    );
-    if (codeScansViaLs) {
-      await this.contextService.setContext(SNYK_CONTEXT.LS_CODE_PREVIEW, true);
-      Logger.info('Code scans via language server enabled.');
-    } else {
-      await this.contextService.setContext(SNYK_CONTEXT.LS_CODE_PREVIEW, false);
-      Logger.info('Code scans are not using Language Server.');
-    }
-
-    await this.languageServer.start();
-
-    this.codeScanOrchestrator = new CodeScanOrchestrator(
-      this.experimentService,
-      this.languageServer,
-      Logger,
-      this.contextService,
-      this,
-    );
+    await this.contextService.setContext(SNYK_CONTEXT.LS_CODE_PREVIEW, true);
+    const lsStartupTask = this.languageServer.start();
```

Comment:
suggestion: rename this to `asyncLsStartupTask || lsStartupTaskPromise`. Common practice is to indicate something as async, usually by adding something like an `async` prefix or `Promise` suffix